scripts/install.sh

```diff
@@ -24,3 +24,5 @@ npm install -g stylelint-config-prettier@5.2.0
 npm install -g stylelint-config-css-modules@1.4.0
 npm install -g prettier@1.18.2
 npm install -g stylelint-prettier@1.1.1
+npm install -g stylelint-config-styled-components@0.1.1
+npm install -g stylelint-processor-styled-components@1.5.2
```

Comment:
Why did you choose this version instead of the latest 1.8.0 ?

Reply:
Its the latest one we use in our projects and I know that it works with 1.5.2. If its easier for you I can test it with 1.8.0 and change the PR accordingly.

Reply:
Yes, it could be cool :)